### ISSUE TITLE
How can I update the Node version with mupx?

### ISSUE THREAD

**Issue by micahalcorn** [arunoda/meteor-up-legacy#984]
The mupx-specific README suggests updating mupx itself, which I have done. It doesn't mention the use of `"nodeVersion"` in the mup.json, so is that still appropriate? I am trying to run `DEBUG=* mupx setup` and there is no verbose output (simply Installing Docker, success, Setting up environment, success). 

I'm trying to update so that I can try to deploy with bcrypt imported in Meteor 1.3 or wrapped using `meteor add npm-bcrypt`. I keep getting `error: this app is not compatible with architecture 'os.linux.x86_64'`, but again, I can't see any verbose output.

**Comment by MasterJames** [comment]
Updating meteor should now more recently give you 1.3.2.2 which solves various fiber problems (npm module building etc.), and is highly recommended. Although there might be a 1.3.2.4 now too?
I believe nodejs is preset in the docker containers and not as much an issue with mupx setup.

The new version of mupx is now porting over at kadirahq/meteor-up were the devs there will be more help with newer release issues.

**Comment by micahalcorn** [comment]
Thanks @MasterJames, I'm running my app fine with bcrypt locally. It's only the deployment process that is affected, so I don't think it is related to my Meteor version (I've reproduced the issue in 1.3.2.4). I'm unclear on how to even confirm which version of Node my Docker container is running. I also tried using the new meteor-up package with the new `mup deploy` command and I have the same problems.

**Comment by MasterJames** [comment]
Google is good for that sort of thing.
http://stackoverflow.com/questions/12014750/how-can-i-know-the-version-of-node-being-used-by-my-meteor-app

**Comment by micahalcorn** [comment]
I was under the impression that the app could be running with a different node version in the docker container, and I couldn't figure out how to confirm that.

Anyway, I seem to have been mistaken on my previous comment about attempting to upgrade Meteor. I made a ton of changes to my environment and then tried again to upgrade to Meteor 1.3.2 and it seems to have resolved the problem of architecture compatibility. Separately I discovered that the bcrypt node module installed on my local machine (OS X) cannot run on my Ubuntu server, so I've resorted to using the wrapped Meteor package instead of using `import`. Thanks for the insights.
